src/devices/Hx711/README.md

```diff
@@ -0,0 +1,141 @@
+﻿# HX711: Load Cell amplifier and 24-Bit ADC Module
+
+## Summary
+
+Device binding for the HX711 weight componet. This componet is necessary if you want to read the weight from a load cell, due to the analog signal and too weak emitted.
+The Load Cell Amplifier and ADC Module is a small breakout board for the HX711 IC that allows you to easily read load cells to measure weight. 
+By connecting the module to your microcontroller you will be able to read the changes in the resistance of the load cell and with some calibration you’ll be able to get very accurate weight measurements. 
+This can be handy for creating your own industrial scale, process control, or simple presence detection.
+
+## Documentation
+
+**[HX711 - Source 1](https://homotix_it.e-mind.it/upld/repository/File/hx711.pdf)**
+
+**[HX711 - Source 2](https://html.alldatasheet.com/html-pdf/1132222/AVIA/HX711/457/1/HX711.html)**
+
+**[Example for Arduino](https://www.moreware.org/wp/blog/2020/06/09/arduino-come-funziona-la-board-per-celle-di-carico-hx711/)**
+
+## Usage with calibration process
+```C#
```

Comment:
you're missing an extra enter before the code block.
And the name of the block should be `csharp`

Reply:
Ok, i fixed it